### ISSUE TITLE
]

### ISSUE THREAD

**Issue by shockergit** [koreader/koreader#7550]
**Does your feature request involve difficulty completing a task? Please describe.**
A clear and concise description of what the problem is. Ex. I think it takes too many steps to [...]

**Describe the solution you'd like**
A clear and concise description of what you'd like to happen.

**Describe alternatives you've considered**

**Additional context**
Add any additional context or screenshots about the feature request here.

**Comment by poire-z** [comment]
Yep, out of scope, we're a reader, not an editor. And the amount of UI needed to have this working would be huge...
